admin/AmazonAI-PollyConfiguration.php

```diff
@@ -411,9 +411,19 @@ function sort_voices( $voice1, $voice2 ) {
 				return strcmp( $voice1['LanguageName'], $voice2['LanguageName'] );
 		}
 
+        //TODO: add more languages according to the Source language in general settings
+        $voice_list = array(
```

Comment:
can you explain why this is needed?

Reply:
thats the hardcoded part we talked about. 
we can set a sergio endpoint to return the relevant voices. 

@borisjaskerovich  - not all voices listed here

Reply:
@royasoolin basically yes, but why not allow all the voices available by AWS? 
sergio should support it, no ?

Reply:
@shlomisut it can support it, right now it supports just female/male for the 5 languages we decided to support

Reply:
yes, I mean supporting voice in sergio API is rather simple, and persistence already supports it